CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ BACKWARDS INCOMPATIBILITIES:
 * compute: `google_compute_instance`, `google_compute_instance_from_template` `metadata` field is now authoritative and will remove values not explicitly set in config. [GH-2208]
 * compute: `google_compute_project_metadata` resource is now authoritative and will remove values not explicitly set in config. [GH-2205]
 * compute: `google_compute_url_map` resource is now authoritative and will remove values not explicitly set in config. [GH-2245]
+* compute: `google_compute_global_forwarding_rule.labels` is removed from the `google` provider and must be used in the `google-beta` provider. [GH-2399]
```

Comment:
why did this happen???

Reply:
Huh! That was weird. I expect it was a race condition between the Magician doing merge-prs and my changes to the changelog? Looks like another run clobbered it.